main/guides/js-programming/far.md

```diff
@@ -28,13 +28,39 @@ In particular, note that every object returned from a smart contract, such a `pu
 `creatorFacet`, must be passable. All objects used in your contract's external API must
 be passable.
 
+### Rules for creating passable objects
+Record and tuples are hardened acyclical objects which can be passed-by-copy if they satisfy some constraints.
+
+#### Record
+A plain object where:
+- All property values must be passables.
+  - Properties cannot be accessors.
+- All properties must be own and enumerable, with string names.
+  - Symbols are not allowed.
+  - The prototype must be `Object.prototype` or `null`.
+- Nested records and tuples cannot form a cycle.
+- It must be hardened with `harden()`.
+
+#### Tuple
+A plain array where:
+- All property values must be passables.
+  - Properties cannot be accessors.
+- All properties must be own, and be all the integers indexes or `length`.
+  - The array cannot have holes.
+  - The prototype must be `Array.prototype`.
+- Nested records and tuples cannot form a cycle.
+- It must be hardened with `harden()`.
+
 ### Rules for creating remotables
 - All property values must be functions. 
   - They cannot be accessors.
 - You must wrap the object with `Far()`.
 
 **Note**: ERTP objects, such as `Purses`, are automatically created as `Remotable`, as are
-`UserSeats` and `ZCFSeats`. 
+`UserSeats` and `ZCFSeats`.
```

Comment:
UserSeat should link to https://agoric.com/documentation/zoe/api/zoe.html#userseat-object
ZCFSeat should link to https://agoric.com/documentation/zoe/api/zoe-contract-facet.html#zcfseat-object
Purse to https://agoric.com/documentation/ertp/api/purse.html